R-package/R/utils.R

```diff
@@ -235,33 +235,34 @@ convert.labels <- function(labels, objective_name) {
 }
 
 # Generates random (stratified if needed) CV folds
-generate.cv.folds <- function(nfold, nrows, stratified, label, params) {
+generate.cv.folds <- function(nfold, nrows, stratified, label, group, params) {
+  if (NROW(group)) {
+    return(generate.group.folds(nfold, group))
```

Comment:
It appears that we don't support stratified CV when `group` is non-empty. Can we throw a warning about the use of un-stratified splitting when `group` is non-empty?

Reply:
Changed it to throw an error rather than a warning.

Reply:
Changed it to a warning now.